hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/aws_sdk_v2_changelog.md

```diff
@@ -0,0 +1,105 @@
+  `ExecutionInterceptor`
+* "Registering" a Span with a request has moved from `requestCreated` to `beforeExecution` 
+  (where an `ExecutionAttributes` instance is first available)
+* The ReferrerHeader is built and added to the http request in `modifyHttpRequest`,
+  rather than in `beforeExecution`, where no http request is yet available
+* Dynamic loading of interceptors has been implemented to reproduce previous behaviour
+  with `RequestHandler2`s. The AWS SDK v2 offers an alternative mechanism, described
+  [here](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/interceptor/ExecutionInterceptor.html)
+  under "Interceptor Registration", which could make it redundant.
+
+In the Transfer Manager, `TransferListener` replaces `TransferStateChangeListener`. S3A code
+has been updated, but registration of the new listeners is currently commented out because 
+it causes an incompatibility issue with the internal logger, resulting in `NoSuchMethodError`s,
+at least in the current TransferManager Preview release. 
+
+
+## Metric Collection
+
+`AwsStatisticsCollector` has been updated to implement the new `MetricPublisher` interface
+and collect the metrics from a `MetricCollection` object.
+The following table maps SDK v2 metrics to their equivalent in v1:
+
+| v2 Metrics	                                                 | com.amazonaws.util.AWSRequestMetrics.Field	 | Comment	                       |
+|-------------------------------------------------------------|---------------------------------------------|--------------------------------|
+| CoreMetric.RETRY_COUNT	                                     | HttpClientRetryCount	                       | 	                              |
+| CoreMetric.RETRY_COUNT	                                     | RequestCount	                               | always HttpClientRetryCount+1	 |
+| HttpMetric.HTTP_STATUS_CODE with HttpStatusCode.THROTTLING	 | ThrottleException	                          | to be confirmed	               |
+| CoreMetric.API_CALL_DURATION	                               | ClientExecuteTime	                          | 	                              |
+| CoreMetric.SERVICE_CALL_DURATION	                           | HttpRequestTime	                            | 	                              |
+| CoreMetric.MARSHALLING_DURATION	                            | RequestMarshallTime	                        | 	                              |
+| CoreMetric.SIGNING_DURATION	                                | RequestSigningTime	                         | 	                              |
+| CoreMetric.UNMARSHALLING_DURATION	                          | ResponseProcessingTime	                     | to be confirmed	               |
+
+Note that none of the timing metrics (`*_DURATION`) are currently collected in S3A.
+
+
+## DeleteObject
+
+In SDK v2, bulk delete does not throw on partial failure: the response contains a list of errors. 
```

Comment:
Could you also mention what previous behaviour was, just to make clear what we've changed.